btcnode.adoc

```diff
@@ -0,0 +1,95 @@
+
+=== Slack
+
+Discussion of Bitcoin Core node status and updates takes place in the `#bisq-btcnode` Slack channel.
+
+
+== Roles
+
+
+=== Maintainer
+
+The contributor(s) responsible for full node <<infrastructure>> and <<process>>.footnote:[See link:roles.html#maintainer[]]
+
+==== Role Issue
+
+{gh-org}/roles/issues/66[bisq-network/roles#66] footnote:[See link:roles.html#issue[]]
+
+==== Role Team
+:btcnode-maintainers: {gh-team}/btcnode-maintainers[@bisq-network/btcnode-maintainers]
+
+{btcnode-maintainers} footnote:[See link:roles.html#team[]]
+
+==== Duties
+
+ * Monitor communications on the `#bisq-btcnode` Slack channel.footnote:[See link:roles.html#communication[]]
+ * Keep this documentation up to date (in particular, the <<process>> detailed below).footnote:[See link:roles.html#documentation[]]
+ * Write a monthly report on the Bitcoin Node Maintainer <<role-issue>>.footnote:[See link:roles.html#reporting[]]
+
+==== Rights
+
+ * Write access to the {gh-org}/bisq-btcnode[bisq-network/bisq-btcnode] repository.
+
+=== Operator
+
+The contributor(s) responsible for keeping full nodes running as outlined in <<process>>.footnote:[See link:roles.html#maintainer[]]
+
+==== Role Issue
+
+{gh-org}/roles/issues/67[bisq-network/roles#67] footnote:[See link:roles.html#issue[]]
+
+==== Role Team
+:btcnode-operators: {gh-team}/btcnode-operators[@bisq-network/btcnode-operators]
+
+{btcnode-operators} footnote:[See link:roles.html#team[]]
+
+==== Duties
+
+ * Operate bitcoin nodes as detailed below in <<process>>.
+ * Monitor communications on the `#bisq-btcnode` Slack channel.footnote:[See link:roles.html#communication[]]
+ * Write a monthly report on the Bitcoin Node Operator <<role-issue>>.footnote:[See link:roles.html#reporting[]]
+
+==== Rights
+
+ * N/A
+
+
+== Process
+
+[ the following is a rough outline in need of feedback and with blanks to fill ]
+
+=== Step 0. Evaluate
+
+Determine if this is really a role you can fill. It's expected that your node will maintain (1) good uptime and (2) an up-to-date config and bitcoind version (as required by the Bitcoin Node Maintainer). There are no strict requirements for either, but it's expected that you will make your best effort to stay current.
+
+=== Step 1. Get in Touch
+
+Contact the Bitcoin Core Maintainer to convey your intention of setting up a Bitcoin Core node for Bisq.
+
+=== Step 2. Set it Up
+
+Set up a Bitcoin Core node, making sure to use the configuration https://github.com/bisq-network/bisq-btcnode[specified here] [where is it, actually?]. Also make sure your node runs in a secure environment [ details? ] and that it allows onion connections so Bisq can connect to it.
+
+=== Step 3. Include Node in Bisq Source
+
+Once your node is ready, send a PR to include it as a Bitcoin Core node for Bisq (they're currently specified https://github.com/bisq-network/bisq/blob/ff4e22d5f537ec8e77bffb3fec116b96fa387c46/core/src/main/java/bisq/core/btc/nodes/BtcNodes.java[here]).
```

Comment:
Nit: probably better to make links like this refer to `master` vs. a specific commit, just so when they click the link, they get an up-to-date view on things. The downside of course is that if that class is moved or renamed, the link will not work anymore.

Reply:
Thanks for noticing...not sure how that happened.